src/averageAges.js

```diff
@@ -14,12 +14,31 @@
  *
  * @return {number}
  */
+const MALE_SEX = 'm';
+const FEMALE_SEX = 'f';
+const CENTURY_VALUE = 100;
+
+function calculateAverage(arrayPeople) {
+  return arrayPeople.reduce((sumAges, person) => {
+    const age = person.died - person.born;
+
+    return sumAges + age;
+  }, 0) / arrayPeople.length;
```

Comment:
it would be better if you save result of reduce in a variable, and then use it with return and / arrayPeople.length, gonna be more readable